docs/reference/adventureworks/armTemplates/auxiliary/policies.json

```diff
@@ -18097,6 +18097,1275 @@
+                              "storageAccountAccessKey": "[[listkeys(parameters('vulnerabilityAssessmentsStorageID'), providers('Microsoft.Storage', 'storageAccounts').apiVersions[0]).keys[0].value]",
+                              "recurringScans": {
+                                "isEnabled": true,
+                                "emailSubscriptionAdmins": false,
+                                "emails": [
+                                  "[[parameters('vulnerabilityAssessmentsEmail')]"
+                                ]
+                              }
+                            }
+                          }
+                        ],
+                        "outputs": {}
+                      },
+                      "parameters": {
+                        "location": {
+                          "value": "[[field('location')]"
+                        },
+                        "synapseWorkspaceName": {
+                          "value": "[[first(split(field('fullname'),'/'))]"
+                        },
+                        "synapseSqlPoolName": {
+                          "value": "[[field('name')]"
+                        },
+                        "vulnerabilityAssessmentsEmail": {
+                          "value": "[[parameters('vulnerabilityAssessmentsEmail')]"
+                        },
+                        "vulnerabilityAssessmentsStorageID": {
+                          "value": "[[parameters('vulnerabilityAssessmentsStorageID')]"
+                        },
+                        "vulnerabilityAssessmentsStorageContainerName": {
+                          "value": "[[parameters('vulnerabilityAssessmentsStorageContainerName')]"
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "name": "Deploy-Synapse-Sql-VulnerabilityAssessment"
+        },
+        {
+          "properties": {
+            "displayName": "Deploy-Synapse-VulnerabilityAssessment",
+            "policyType": "Custom",
+            "mode": "Indexed",
+            "description": "Deploy Synapse workspace vulnerability assessments.",
+            "metadata": {
+              "version": "1.0.0",
+              "category": "Synapse"
+            },
+            "parameters": {
+              "effect": {
+                "type": "String",
+                "metadata": {
+                  "displayName": "Effect",
+                  "description": "Enable or disable the execution of the policy"
+                },
+                "allowedValues": [
+                  "DeployIfNotExists",
+                  "Disabled"
+                ],
+                "defaultValue": "DeployIfNotExists"
+              },
+              "vulnerabilityAssessmentsEmail": {
+                "type": "String",
```

Comment:
Should this be an Array to allow multiple emails?

Reply:
This should be consistent with other SQL policies.